### PR TITLE
Expand `Audio Editing`

### PR DESCRIPTION
**Pull request by FazlyMR**
This is my second pull request of the day for this guide, as the first one was closed soon after realizing I had forgotten to outdate existing translations of this guide.

There are several updates made in this commit, these are:
1. Mentions on the inclusion of the LAME encoding library after the expiration of the last few patents on the MP3 audio format in 2017. As reference, the [Audacity Reference Manual](https://manual.audacityteam.org/man/faq_installing_the_lame_mp3_encoder.html) and the [Wikipedia article](https://en.wikipedia.org/wiki/MP3#Licensing,_ownership,_and_legislation) on the MP3 audio format also refers to the above fact.
3. Mentions another guide, "Compressing files", for more information on compression in general.
4. Revised Audacity instructions to skip installing LAME (now built-in) and match audio export settings to the previously mentioned guide.
5. Revised mp3DirectCut description.

Note: On the outdate commit, it refers to a merge request number in my fork and not this fork. Please ignore it.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Guides/Audio_Editing/en.md [hunk 1]
```diff
@@ -8,27 +8,32 @@ This article serves as guide to help you do minor edits to your audio files for
 
 ## Audacity (and LAME)
 
-[Audacity](https://www.audacityteam.org/download) is a open source free audio editing and recording software. To properly use this for `.mp3`, you will need to use LAME.
+[Audacity](https://www.audacityteam.org/download) is a free, open-source audio editing and recording software. It uses the [LAME](https://lame.sourceforge.io) encoding library to be able to export audio in the MP3 audio format.
 
-[LAME](https://lame.sourceforge.io) is an `.mp3` encoding library that will allow Audacity to export sound files in the `.mp3` format whilst using different bit rates. To install LAME on Audacity, refer to the [Audacity wiki](https://manual.audacityteam.org/man/faq_installing_the_lame_mp3_encoder.html).
+Since 2017, after the last few patents on the MP3 audio format had expired, the LAME encoding library started to be included in later versions of Audacity on Windows and macOS, thus manual installation is no longer necessary.
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:13]: this bit of history is a bit off topic for the wiki

File: wiki/Guides/Audio_Editing/en.md [hunk 2]
```diff
@@ -8,27 +8,32 @@ This article serves as guide to help you do minor edits to your audio files for
...
+Install and open Audacity then follow these steps:
 
 1. Open the `.mp3` file that you want to lower the bit rate on.
-2. Press `Ctrl` + `Shift` + `E`, or
-   1. Click `File`.
-   2. Click `Export Audio...`.
-3. Change "Save as type:" to `MP3 Files`
-4. In the "Format Options", click on `Average` as the bit rate mode.
-5. Navigate to the location you want to save the file as.
+2. Press `Ctrl` + `Shift` + `E`, then change "Save as type:" to `MP3 Files` or:
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:24]: code block for all ui text

File: wiki/Guides/Audio_Editing/en.md [hunk 3]
```diff
@@ -8,27 +8,32 @@ This article serves as guide to help you do minor edits to your audio files for
...
+   1. Click `File`, then `Export` and `Export as MP3`.
+3. In the "Format Options", change the following settings:
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:26]: "In `Format Options`, change ..."

etc

File: wiki/Guides/Audio_Editing/en.md [hunk 4]
```diff
@@ -44,18 +49,19 @@ Install Audacity and LAME, open Audacity then follow these steps:
...
+10. Navigate to the location you want to save the file as.
     - You could rename the file too.
-12. Click `Save`.
+11. Click `Save` and a dialog will appear to enter metadata.
+12. Click `OK` once done entering metadata.
 
 ### Cropping
 
-Install Audacity and LAME, open Audacity then follow these steps:
+Install and open Audacity then follow these steps:
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:64]:
```suggestion
Install and open Audacity, then follow these steps:
```

File: wiki/Guides/Audio_Editing/en.md [hunk 5]
```diff
@@ -8,27 +8,32 @@ This article serves as guide to help you do minor edits to your audio files for
 
 ## Audacity (and LAME)
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:9]:
```suggestion
## Audacity
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Guides/Audio_Editing/en.md (hunk 1 above)
Comment (FazlyMR) [reply wiki/Guides/Audio_Editing/en.md:13]: I think you're right, maybe just make it shorter to at least say that LAME is built into Audacity on Windows and macOS, so the next paragraph for Linux users would make sense.

Comment (FazlyMR) [reply wiki/Guides/Audio_Editing/en.md]: Or maybe just delete it outright, I'm not sure which is best so maybe up to you?

File: wiki/Guides/Audio_Editing/en.md [hunk 6]
```diff
@@ -6,29 +6,32 @@ This article serves as guide to help you do minor edits to your audio files for
...
 
-## Audacity (and LAME)
+## Audacity
 
-[Audacity](https://www.audacityteam.org/download) is a open source free audio editing and recording software. To properly use this for `.mp3`, you will need to use LAME.
+[Audacity](https://www.audacityteam.org/download) is a free, open-source audio editing and recording software. It uses the [LAME](https://lame.sourceforge.io) encoding library to be able to export audio in the MP3 audio format and is built into Audacity on Windows and macOS.
 
-[LAME](https://lame.sourceforge.io) is an `.mp3` encoding library that will allow Audacity to export sound files in the `.mp3` format whilst using different bit rates. To install LAME on Audacity, refer to the [Audacity wiki](https://manual.audacityteam.org/man/faq_installing_the_lame_mp3_encoder.html).
+Linux users should refer to the following article on the [Audacity Reference Manual](https://manual.audacityteam.org/man/installing_and_updating_audacity_on_linux.html#linlame) for more information, as certain Linux distributions may still not provide LAME when installing Audacity but most do provide it.
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:13]:
```suggestion
Linux users should refer to the [Audacity Reference Manual](https://manual.audacityteam.org/man/installing_and_updating_audacity_on_linux.html#linlame) for more information, as certain Linux distributions may not provide LAME when installing Audacity, but most do provide it.
```

File: wiki/Guides/Audio_Editing/en.md [hunk 7]
```diff
@@ -6,29 +6,32 @@ This article serves as guide to help you do minor edits to your audio files for
...
 
 ### Lowering Bit Rate
 
-Install Audacity and LAME, open Audacity then follow these steps:
+*For information on compression in general, see: [Compressing files](/wiki/Guides/Compressing_files)*
+
+Install and open Audacity then follow these steps:
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:19]:
```suggestion
Install and open Audacity, then follow these steps:
```

File: wiki/Guides/Audio_Editing/en.md [hunk 8]
```diff
@@ -44,18 +47,19 @@ Install Audacity and LAME, open Audacity then follow these steps:
...
-   1. Click `File`.
-   2. Click `Export Audio...`.
-9. Change "Save as type:" to `MP3 Files`
-10. In the "Format Options", click on `Average` as the bit rate mode.
-11. Navigate to the location you want to save the file as.
+8. Press `Ctrl` + `Shift` + `E`, then change `Save as type:` to `MP3 Files` or:
+   1. Click `File`, then `Export` and `Export as MP3`.
+9. In the `Format Options`, change the following settings: 
+   1. `Preset` as the bit rate mode
+   2. `Medium, 145-185 kbps` as the quality.
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:54]:
```suggestion
8. Press `Ctrl` + `Shift` + `E`, then in `Save as type:` select `MP3 Files`, or:
   1. Click `File`, then `Export`, then `Export as MP3`.
9. In the `Format Options`, change the following settings: 
   1. `Bit Rate Mode`: `Preset`
   2. `Quality`: `Medium, 145-185 kbps`
```

File: wiki/Guides/Audio_Editing/en.md [hunk 9]
```diff
@@ -64,19 +68,19 @@ Install Audacity and LAME, open Audacity then follow these steps:
...
-8. Change "Save as type:" to `MP3 Files`
-9. In the "Format Options", click on `Average` as the bit rate mode.
-10. Navigate to the location you want to save the file as.
-    - You could rename the file too.
-11. Click `Save`.
+7. Press `Ctrl` + `Shift` + `E`, then change `Save as type:` to `MP3 Files` or:
+   1. Click `File`, then `Export` and `Export as MP3`.
+8. In the `Format Options`, change the following settings: 
+   1. `Preset` as the bit rate mode
+   2. `Medium, 145-185 kbps` as the quality.
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:75]:
```suggestion
7. Press `Ctrl` + `Shift` + `E`, then in `Save as type:` select `MP3 Files`, or:
   1. Click `File`, then `Export`, then `Export as MP3`.
8. In the `Format Options`, change the following settings: 
   1. `Bit Rate Mode`: `Preset`
   2. `Quality`: `Medium, 145-185 kbps`
```

File: wiki/Guides/Audio_Editing/en.md [hunk 10]
```diff
@@ -6,29 +6,32 @@ This article serves as guide to help you do minor edits to your audio files for
...
-   1. Click `File`.
-   2. Click `Export Audio...`.
-3. Change "Save as type:" to `MP3 Files`
-4. In the "Format Options", click on `Average` as the bit rate mode.
-5. Navigate to the location you want to save the file as.
+2. Press `Ctrl` + `Shift` + `E`, then change `Save as type:` to `MP3 Files` or:
+   1. Click `File`, then `Export` and `Export as MP3`.
+3. In the `Format Options`, change the following settings: 
+   1. `Preset` as the bit rate mode
+   2. `Medium, 145-185 kbps` as the quality.
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:26]:
```suggestion
2. Press `Ctrl` + `Shift` + `E`, then in `Save as type:` select `MP3 Files`, or:
   1. Click `File`, then `Export`, then `Export as MP3`.
3. In the `Format Options`, change the following settings: 
   1. `Bit Rate Mode`: `Preset`
   2. `Quality`: `Medium, 145-185 kbps`
```

File: wiki/Guides/Audio_Editing/en.md [hunk 11]
```diff
@@ -64,19 +68,19 @@ Install Audacity and LAME, open Audacity then follow these steps:
...
+9. Navigate to the location you want to save the file as.
+   - You could rename the file too.
+10. Click `Save` and a dialog will appear to enter metadata.
+11. Click `OK` once done entering metadata.
 
 ## mp3DirectCut
 
-[mp3DirectCut](https://mpesch3.de) is a free to use audio editing software.
-This tool is recommended when you only want to raise/lower the volume or crop the audio as it doesn't require you to re-encode the audio, meaning less quality loss.
+[mp3DirectCut](https://mpesch3.de) is a free-to-use audio editing software that can directly edit MP3 files without re-encoding which could result in less quality loss. It is recommended when needing to raise/lower the volume or crop the audio.
```
Comment (Walavouchey) [inline wiki/Guides/Audio_Editing/en.md:83]:
```suggestion
[mp3DirectCut](https://mpesch3.de) is a free-to-use audio editing software that can directly edit MP3 files without re-encoding, often preventing loss of quality. It is recommended when needing to raise/lower the volume or crop the audio.
```

File: wiki/Guides/Audio_Editing/en.md (hunk 6 above)
Comment (FazlyMR) [reply wiki/Guides/Audio_Editing/en.md:13]: Forgot to do that.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**